CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Fixed: `function-url-quotes` now ignores spaces within `url()`.
 - Fixed: `value-list-comma-*` rules now ignore SCSS maps.
 - Fixed: `no-descending-specificity` now ignores trailing colons within selectors.
+- Fixed: `function-comma-newline-after` now allows comments at the end of a line.
```

Comment:
Let's saw "allows end-of-line comments." That would be consistent with the similar changes in the changelog.